src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -92057,4 +92038,78 @@ SELECT TAB_MOLECULE_PARAMETERS.MOLECULE AS MoleculeName,

+WHERE  [tab_container_parameter_rates].[calculation_method] = [tab_species_calculation_methods].[calculation_method]

+         AND [tab_container_parameters].[container_id] = [tab_container_parameter_rates].[container_id]

+         AND [tab_container_parameters].[container_type] = [tab_container_parameter_rates].[container_type]

+         AND [tab_container_parameters].[container_name] = [tab_container_parameter_rates].[container_name]

+         AND [tab_container_parameters].[parameter_name] = [tab_container_parameter_rates].[parameter_name]

+         AND [tab_container_parameters].[building_block_type] = "INDIVIDUAL"

+GROUP  BY

+          [tab_container_parameter_rates].[container_id], 

+          [tab_container_parameter_rates].[container_type], 

+          [tab_container_parameter_rates].[container_name], 

+          [tab_container_parameter_rates].[parameter_name], 

+          [tab_container_parameter_rates].[calculation_method], 

+          [tab_container_parameter_rates].[formula_rate]

+HAVING COUNT ([tab_species_calculation_methods].[species]) = (SELECT COUNT ([species])

+       FROM   [tab_species])

+UNION

+SELECT DISTINCT 

+                [tab_container_parameter_values].[container_id] AS [ContainerId], 

+                [tab_container_parameter_values].[container_type] AS [ContainerType], 

+                [tab_container_parameter_values].[container_name] AS [ContainerName], 

+                [tab_container_parameter_values].[parameter_name] AS [ParameterName], 

+                0 AS [IsSameFormula]

+FROM   [tab_container_parameters],

+       [tab_container_parameter_values],

+       [tab_species_parameter_value_versions]

+WHERE  [tab_container_parameters].[container_id] = [tab_container_parameter_values].[container_id]

+         AND [tab_container_parameters].[container_type] = [tab_container_parameter_values].[container_type]

+         AND [tab_container_parameters].[container_name] = [tab_container_parameter_values].[container_name]

+         AND [tab_container_parameters].[parameter_name] = [tab_container_parameter_values].[parameter_name]

+         AND [tab_species_parameter_value_versions].[species] = [tab_container_parameter_values].[species]

+         AND [tab_species_parameter_value_versions].[parameter_value_version] = [tab_container_parameter_values].[parameter_value_version]

+         AND [tab_container_parameters].[building_block_type] = "INDIVIDUAL"

+GROUP  BY

+          [tab_container_parameter_values].[container_id], 

+          [tab_container_parameter_values].[container_type], 

+          [tab_container_parameter_values].[container_name], 

+          [tab_container_parameter_values].[parameter_name], 

+          [tab_container_parameter_values].[default_value]

+HAVING COUNT ([tab_container_parameter_values].[species]) = (SELECT COUNT ([species])

+       FROM   [tab_species])

+ORDER  BY

+          [IsSameFormula],

+          [ParameterName],

+          [ContainerId];

```

Comment:
This is the new vie for all parameters with BB-Type "INDIVIDUAL" which have either the same formula or the same value for all species.
IT is a `UNION ` query with 2 parts:
* Part 1 - select all container parameters with the same **formula**
  1. group all parameters of BB-Type "INDIVIDUAL" which are defined by a formula in _tab_container_parameter_rates_ by `{Container, Parameter, Calculation Method, Formula_Rate}`
  2. for each such group: count the species in _tab_species_calculation_methods_ which are defined for the CalcMethod ofthe group)
  3. if the number of species for the group = number of species defined in _tab_species_: `{Container, Parameter}` is described by the same combination of `{Calculation Method, Formula_Rate}` for ALL species and thus has the same formula of all species
* Part 2 - select all container parameters with the same **value**
  1. group all parameters of BB-Type "INDIVIDUAL" which are defined by a value in _tab_container_parameter_values_ by `{Container, Parameter, Value}`
  2. if the number of species in _tab_container_parameter_values_ for the group = number of species defined in _tab_species_: `{Container, Parameter}` has the same value of all species